### DISCUSSION TITLE
Specific Zone Selection for Work and School Tour

### DISCUSSION THREAD

**Discussion by HuyLeDinhGia** [synthicity/activitysim#913] [Q&A] [answered]
Hi Dev Team,
I'm using example estimation and test prototype mtc models. In my data there is CBD zone which is especially attractive and most of work tours destinations are CBD, and It is hard to calibrate the coefficients to match the ground truth. How can I adjust this by adding some specific coefficients for CBD zone to the modules (work location, school location, ...)?

Regards,
Huy

**Comment by jfdman** [comment, marked as answer]
Hi @HuyLeDinhGia, you can add a numeric variable in your input zone data that identifies this zone and add a positive coefficient to your size term file to make it more attractive.
